src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -80,6 +79,34 @@ class OppgaveService(
             else -> error("Kunne ikke finne riktig BehandlingType for oppfølgingsoppgave")
         }
 
+    private fun finnAktuellMappeForFremleggsoppgave(enhetsnummer: String?, oppgavetype: Oppgavetype): Long? {
+        if (enhetsnummer == "4489" && oppgavetype == Oppgavetype.Fremlegg) {
+            val mapper = finnMapper(enhetsnummer)
+            val mappeIdForFremleggsoppgave = mapper.find { it.navn.contains("41 - Revurdering") }?.id?.toLong()
+            mappeIdForFremleggsoppgave?.let {
+                logger.info("Legger oppgave i Revurdering vedtak-mappe")
+            } ?: run {
+                logger.error("Fant ikke mappe for fremleggsoppgave: 41 - Revurdering for enhetsnummer=$enhetsnummer")
```

Comment:
Kanskje greit å få med oppgaveId for å lettere debugge hvis det skjer en feil

Reply:
Logger behandlingId